server/src-lib/Hasura/RQL/Types/SchemaCache.hs

```diff
@@ -774,7 +788,12 @@ getDependentObjsWith f sc objId =
     isDependency deps = not $ HS.null $ flip HS.filter deps $
       \(SchemaDependency depId reason) -> objId `induces` depId && f reason
     -- induces a b : is b dependent on a
-    induces (SOTable tn1) (SOTable tn2)      = tn1 == tn2
-    induces (SOTable tn1) (SOTableObj tn2 _) = tn1 == tn2
-    induces objId1 objId2                    = objId1 == objId2
+    induces (SOTable tn1) (SOTableObj tn2 _)               = tn1 == tn2
```

Comment:
pls confirm that `induces (SOTable tn1) (SOTable tn2)      = tn1 == tn2` is actually redundant and hence removed here.